### ISSUE TITLE
OhMyLearning : Centralised Hub for Learning Resources

### ISSUE THREAD

**Issue by shravan20** [open-source-ideas/open-source-ideas#251]
## Project description

A GitHub Repository, which acts as Centralised Gateway to one of the finest and best space for resources for any beginner to learn from. The repository will learning resources and materials for a beginner to advanced learner to learn from any domain and stack. 

## Relevant Technology

Technology used : **Markdown**, a light weight markup language. 

Existing System : There are definitely several repositories providing this. To get them accumulated in one space is hard. As far as I know they have targeted Tech-Stacks only. I would like to propose a **Centralised Learning Hub for everyone and anyone**, be it development, devops, to marketing, sales, accountancy. 

### Complexity

- [x] Beginner - This project requires no or little prior knowledge of the technolog(y|ies) specified to contribute to the project
- [ ] Intermediate - The user should have some prior knowledge of the technolog(y|ies) to the point where they know how to use it, but not necessarily all the nooks and crannies of the technology
- [ ] Advanced - The project requires the user to have a good understanding of all components of the project to contribute

### Required time (ETA)

- [x] Little work - A couple of days
- [ ] Medium work - A week or two
- [ ] Much work - The project will take more than a couple of weeks and serious planning is required

### Categories

- [ ] Mobile app
- [ ] IoT
- [x] Web app
- [ ] Frontend/UI
- [ ] AI/ML
- [ ] APIs/Backend
- [ ] Voice Assistant
- [ ] Developer Tooling
- [ ] Extension/Plugin/Add-On
- [ ] Design/UX
- [ ] AR/VR
- [ ] Bots
- [ ] Security
- [ ] Blockchain
- [ ] Futuristic Tech/Something Unique

Explanation : ** It is going to be slow process and never ending process. But if we could collaborate people from different domains and learning stacks, we could make it a very useful content and resource hub. **

**Comment by KaKi87** [comment]
For everything related to high-tech and maybe more, the [awesome learning resources](https://github.com/lauragift21/awesome-learning-resources) list and the [awesome](https://github.com/sindresorhus/awesome) project in general are the best FLOSS knowledge management resources available out there. 👍

**Comment by shravan20** [comment]
> For everything related to high-tech and maybe more, the [awesome learning resources](https://github.com/lauragift21/awesome-learning-resources) list and the [awesome](https://github.com/sindresorhus/awesome) project in general are the best FLOSS knowledge management resources available out there.

This is not just confined to TECH learning only. This covers all other aspects as well, like Marketing and sales, Architecture and Engineering, Communications, Science and Technology, Governmental, Health and Medicine and also Law and Public Policy..

A to Z learning Space 💯

**Comment by KaKi87** [comment]
> This is not just confined to TECH learning only

I know, just saying.

**Comment by KOLANICH** [comment]
@shravan20, the discuesion #53, and especially Metacademy, are highly related. That is not a git repo, though, but it may be possible to redesign Metacademy as a federation of git repos.

**Comment by HamzaAnis** [comment]
A GitHub repository to store the resources would be a good option?

**Comment by shravan20** [comment]
Yes

**Comment by shravan20** [comment]
> A GitHub repository to store the resources would be a good option?

I believe it's the best solution for Resource Management System

**Comment by HamzaAnis** [comment]
Github LFS supports 1 GB of a free storage. It will be sufficient for this?
You can read it here. [LFS](https://docs.github.com/en/free-pro-team@latest/github/managing-large-files/about-storage-and-bandwidth-usage)

**Comment by shravan20** [comment]
> Github LFS supports 1 GB of a free storage. It will be sufficient for this?
> You can read it here. [LFS](https://docs.github.com/en/free-pro-team@latest/github/managing-large-files/about-storage-and-bandwidth-usage)

Yes, we can store the media contents in a drive/cloud and give the link and description in the repository.
